yb-voyager/src/query/queryissue/constants.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
Do we need the prefix `query` in the directory names of `queryissue`, `queryparser` ?

Reply:
the query directory is just better file organization, it does not translate to any form of nesting of packages. If there was indeed some form of nesting,  I agree; it would make sense to remove the "query" prefix. But since these are top-level packages, I wanted to stick to the pattern of keeping the folder name same as the package name.